pkg/agent/openflow/client_windows.go

```diff
@@ -49,3 +53,17 @@ func (c *client) UninstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcP
 	cacheKey := fmt.Sprintf("L%s%s%x", svcIP, protocol, svcPort)
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
 }
+
+func (c *client) InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+	var flows []binding.Flow
+	flows = append(flows, c.serviceLBFlow(groupID, svcIP, svcPort, protocol, affinityTimeout != 0))
+	if affinityTimeout != 0 {
+		flows = append(flows, c.serviceLearnFlow(groupID, svcIP, svcPort, protocol, affinityTimeout))
+	}
+
+	flows = append(flows, c.arpResponderFlow(route.GlobalVirtualGWIP, globalVirtualSVCMAC, priorityHigh, cookie.Service))
```

Comment:
Question - besides in ARP responder flow, I do not see globalVirtualSVCMAC is used in other code? Does it mean we can use any MAC if we want?

Reply:
Yes, this `globalVirtualSVCMAC ` (now `virtualSvcMAC `) mac is a virtual one.